pallets/order-book/src/tests/extrinsics.rs

```diff
@@ -234,15 +234,16 @@ fn should_not_create_order_book_for_user_not_nft_owner() {
                 DEX.into(),
                 order_book_id
             ),
-            E::UserIsNotOwnerOfNft
+            E::UserDoesntHaveNft
         );
     });
 }
 
 #[test]
-fn should_create_order_book_for_nft() {
+fn should_not_create_order_book_for_user_dont_have_nft_even_they_owner() {
```

Comment:
```suggestion
fn should_not_create_order_book_for_nft_owner_without_nft() {
```
Sorry if too nitpicky) just seems more readable